app/services/media_storage/azure_adapter.rb

```diff
@@ -0,0 +1,84 @@
+module MediaStorage
+  class AzureAdapter < AbstractAdapter
+    attr_reader :prefix, :container, :storage_account_name, :get_expiration, :put_expiration, :client, :signer
+    DEFAULT_EXPIRES_IN = 3 # time in minutes, see get_expiry_time(expires_in)
+
+    def initialize(opts={})
+      @storage_account_name = opts[:azure_storage_account]
+      @container = opts[:azure_storage_container] || Rails.env
+      @prefix = opts[:azure_prefix]
```

Comment:
we want the default rails env value not on the container - the container has to be explicitly setup with correct ACL etc.

The prefix - this was a means of namespacing the aws s3 bucket paths for isolation to prod / staging etc. 

https://github.com/zooniverse/panoptes/blob/d0eadcf11d61f4510d92cf73f0c1411104de3ce1/app/services/media_storage/aws_adapter.rb#L7-L8
https://github.com/zooniverse/panoptes/blob/d0eadcf11d61f4510d92cf73f0c1411104de3ce1/config/initializers/storage.rb#L10
https://github.com/zooniverse/panoptes/blob/d0eadcf11d61f4510d92cf73f0c1411104de3ce1/kubernetes/deployment-staging.tmpl#L85